keps/sig-scheduling/3633-matchlabelselectors-to-podaffinity/README.md

```diff
@@ -263,8 +266,36 @@ metadata:
             values:
             - database
         topologyKey: topology.kubernetes.io/zone
-        matchlabelKeys: # ADDED
-        - pod-template-hash
+        matchLabelSelectors: # ADDED
+        - key: pod-template-hash
+          operator: In
+```
+
+#### Story 2
+
+Let's say all Pods on each tenant get `tenant` label via a controller or a manifest management tool like Helm. 
+And, the cluster admin now wants to achieve exclusive 1:1 tenant to domain placement.
```

Comment:
Maybe emphasize a bit that the value of `tenant` is unknown when composing the yaml.

```suggestion
Although the value of `tenant` label is unknown when composing the workload's manifest, the cluster admin still wants to achieve exclusive 1:1 tenant to domain placement.
```